sky/shell/platform/glfw/window_impl.cc

```diff
@@ -61,5 +71,66 @@ void WindowImpl::UpdateViewportMetrics(int width, int height) {
   engine_->OnViewportMetricsChanged(metrics.Pass());
 }
 
+void WindowImpl::DispatchMouseButtonEvent(int button, int action, int mods) {
+  pointer::PointerType type;
+  if (action == GLFW_PRESS) {
+    if (!buttons_) {
+      type = pointer::PointerType::DOWN;
+      glfwSetCursorPosCallback(window_, OnCursorPosChanged);
+    } else {
+      type = pointer::PointerType::MOVE;
+    }
+    buttons_ |= 1 << button;
+  } else if (action == GLFW_RELEASE) {
+    buttons_ &= !(1 << button);
```

Comment:
I think there is a typo here. Shouldn't this be `buttons_ &= ~(1 << button);` if you just want to reset that item in the mask?